lib/buildsystems/pip.rb

```diff
@@ -39,6 +39,9 @@ def self.install
     @pip_cache_dir = `pip cache dir`.chomp
     @pip_cache_dest_dir = File.join(CREW_DEST_DIR, @pip_cache_dir)
 
+    @pip_version = `python3 -c "import pip; print(pip.__version__)"`.chomp
+    @pip_resume_retries = Gem::Version.new(@pip_version.to_s) >= Gem::Version.new('25.1') ? '--resume-retries 5' : ''
```

Comment:
Instead of hardcoding 5, make it configurable with a constant/environment variable?

Reply:
Done.